gemrb/GemRB.cfg.noinstall.sample

```diff
@@ -267,6 +268,30 @@ PluginsPath=./gemrb/plugins/
 #  Input Parameters                                 #
 #####################################################
 
+#####################################################
```

Comment:
Should all be a few lines higher, so you don't split the input section into two.

Reply:
C&P is hard sometimes ...